helm/vitess/templates/NOTES.txt

```diff
@@ -3,6 +3,9 @@
 
 Release name: {{.Release.Name}}
 
+After the init-shard-master jobs complete successfully, you can and should delete the jobs.
```

Comment:
In the Operator, I have a few Jobs that stick around as a record that "this was already done". For example, if you delete them, they will run again every time you `helm upgrade` anyway.

If the Jobs were truly one-off, it would be important to delete them since the number of finished Jobs could grow without bound. However, since we are deterministically creating only one Job per shard, and uninstalling the chart should delete any Jobs we create, I don't think it's harmful to keep the Jobs around.

Note that the Pod GC will clean up the terminated Pods eventually, yet the Job will remember in its status that it already completed.

Reply:
Ok, I'll change the comment to reflect that.